src/CoreLib/DSeq.fs

```diff
@@ -89,23 +90,26 @@ and internal DStreamForwardDependencyType =
     /// Multicast a blob to all destinations in the Mapping
     | MulticastToNetwork of DependentDStream
 and internal DStreamFactory() = 
-    inherit HashCacheFactory<DStream>()
-    static let collectionByName = ConcurrentDictionary<_,_>(StringTComparer<int64>(StringComparer.Ordinal)) 
-    // Retrieve a DStream
-    static member ResolveDStream( hash ) = 
-        DStreamFactory.Resolve( hash ) 
+    static let collectionByJob = ConcurrentDictionary<Guid,(int64 ref)*ConcurrentDictionary<_,_>>() 
```

Comment:
add some comments to explain about the value type  (int64 ref) * ConcurrentDictionary<_,_>, what are they?